src/Sarif.Viewer.VisualStudio.Core/SarifViewerPackage.cs

```diff
@@ -203,12 +215,60 @@ private void SolutionEvents_OnBeforeCloseSolution(object sender, EventArgs e)
         {
             // stop watcher when the solution is closed.
             this.sarifFolderMonitor?.StopWatch();
+
+            var fileSystem = new FileSystem();
+
+            try
+            {
+                fileSystem.FileDelete(Path.Combine(GetDotSarifDirectoryPath(), "scan-results.sarif"));
+            }
+            catch (Exception) { }
```

Comment:
will it cause any issue if fail, 
suggest to add explanation.
e.g.
// Doesn't matter because this delete is best-effort.